src/components/Button.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'; // eslint-disable-line no-unused-vars
+import PropTypes from 'prop-types'
+
+const Button = (props) => {
+  return (
+    <button className={'button'} style={{ backgroundColor: (props.color === 'orange' ? '' : 'lightGray'), width: (props.width === true ? '50%' : '') }}>
```

Comment:
[OPTIONAL]
lift out the styling and just pass it to the button as a variable, its cleaner.
```
const style = {
    width: props.wide ? '50%' : '25%',
    background: props.color ? props.color : defaultColor
  }

return (
    <button style={style} >{props.name}</button>
  )
```
or something similar, we still have click event to add and it already looks packed there.